### PR TITLE
update StatusChangedActors api

### PR DESCRIPTION
**Pull request by deaswang**
Update StatusChangedActors API for add/del/change actors.

### CODE REVIEW

File: api/api_full.go [hunk 1]
```diff
@@ -416,7 +416,7 @@ type FullNode interface {
 	StateAccountKey(context.Context, address.Address, types.TipSetKey) (address.Address, error)
 	// StateChangedActors returns all the actors whose states change between the two given state CIDs
 	// TODO: Should this take tipset keys instead?
-	StateChangedActors(context.Context, cid.Cid, cid.Cid) (map[string]types.Actor, error)
+	StateChangedActors(context.Context, cid.Cid, cid.Cid) (map[string]types.Actor, map[string]types.Actor, map[string]types.Actor, error)
```
Comment (magik6k) [inline api/api_full.go:419]: Multi-return doesn't work over jsonrpc

Comment (deaswang) [reply api/api_full.go]: add ChangedActors struct

**Comment by Stebalien** [comment]
Unfortunately, this has gone stale and no longer applies. Additionally, while the new API is unarguably better, it's unclear if it's better _enough_ to make an API breaking change.

If you'd like to pick this up again, please file an issue and we'll discuss from there. Otherwise, I understand if you've moved on by now.

**Comment by Stebalien** [comment]
Actually, I'll leave this open as a draft. I believe it could go in the "v1" API to avoid breaking changes.

**Pull request closed without merging**